ext/ffi/ffi.stub.php

```diff
@@ -55,7 +55,7 @@ public static function memcpy(FFI\CData $dst, $src, int $size): void {}
     public static function memcmp($ptr1, $ptr2, int $size): ?int {}
 
     /** @prefer-ref $ptr */
-    public static function memset(FFI\CData $ptr, int $ch, int $size): void {}
+    public static function memset(FFI\CData $ptr, int $byte, int $size): void {}
```

Comment:
http://www.cplusplus.com/reference/cstring/memset/ uses `value`, but I understand why you propose `$byte` here. Either makes sense to me.

Reply:
I picked `$byte` here as a more accurate version of `$character`, but just `$value` makes more sense to me.